src/index.html

```diff
@@ -11,7 +11,160 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form"
+      >
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="naming">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          >
```

Comment:
make sure to fix such issues over your code
```suggestion
          <input type="text" name="surname">
```